<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="message_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>메시지 내용</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.토공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.거푸집공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.하수도공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4.철근공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.콘크리트공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6.방수공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7.도장공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8.미장공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9.조적공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10.타일공사</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>11.창호공사</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
    </font>
   </property>
   <property name="text">
    <string>메시지 출력</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>공종 선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
    </font>
   </property>
   <property name="text">
    <string>메시지 지우기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>오늘의 날씨</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>온도 기온 강수확률..                       </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquareRound</family>
    </font>
   </property>
   <property name="text">
    <string>메시지 전송</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
